contracts/DutchExchange.sol

```diff
@@ -1100,7 +1100,67 @@ contract DutchExchange {
             return uint(a);
         }
     }
+    
+    //@dev for quick overview of possible sellerBalances to calculate the possible withdraw tokens
+    //@param auctionSellToken is the sellToken defining an auctionPair
+    //@param auctionBuyToken is the buyToken defining an auctionPair
+    //@param user is the user who wants to his tokens
+    //@returns returns sellbal for all indices for all tokenpairs 
+    function getIndicesWithClaimableTokens(
+        address auctionSellToken,
+         address auctionBuyToken,
+        address user
+    )
+        public
+        view
+        returns(uint[])
+    {
+        uint runningAuctionIndex = getAuctionIndex(auctionSellToken, auctionBuyToken);
+        uint[] memory sellBal = new uint[](runningAuctionIndex+1);
+        for(uint j = 0; j <= runningAuctionIndex; j++){
+            sellBal[j] = sellerBalances[auctionSellToken][auctionBuyToken][j][user];
+        }        
+        return sellBal;
+    }    
+
+    //@dev for quick overview of current sellerBalance for a user
+    //@param auctionSellToken are the sellTokens defining an auctionPair
+    //@param auctionBuyToken are the buyTokens defining an auctionPair
+    //@param user is the user who wants to his tokens
+    function getSellerBalanceOfCurrentAuctions(
+        address[] auctionSellToken,
+        address[] auctionBuyToken,
+        address user
+    )
+        public
+        returns(uint[])
```

Comment:
Should this also be a `view` function?And the next one

Reply:
they should